src/components/transport_manager/src/android/android_transport_adapter.cc

```diff
@@ -203,5 +182,24 @@ std::string AndroidTransportAdapter::GetControlReceiverSocketName() const{
     return "";
 }
 
+ std::string AndroidTransportAdapter::GetTransportName() const
+ {
+             switch(transport_type_) {
+            case AndroidTransportType::BLE:
+                return "Bluetooth Low Energy";
```

Comment:
@VadymLuchko I would recommend using prefixes for easier filtration of log messages in the console:
```suggestion
                return "[IPC][CPP][BLE]";
```
Then, you can use "[IPC]" to filter out all IPC messages from both C++ and Java sides. Or you can use "[IPC][CPP]" to filter out all IPC messages from the C++ part only. Or finally, you can use "[IPC][CPP][BLE]" to filter out all BLE transport messages from the C++ part only.

Reply:
@VadymLuchko for log callbacks in receiver and sender classes, you can also append [SENDER] and [RECEIVER] labels correspondingly